### DISCUSSION TITLE
How to start the application on MacOS?

### DISCUSSION THREAD

**Discussion by Kenny-024** [dosbox-staging/dosbox-staging#3800] [Q&A] [answered]
Hello, when I mount the DOSBox Staging .dmg image I have to drag the 2 terminal shell scripts into a folder. What do they do and can I just run DOSBox by opening the application?

**Comment by johnnovak** [comment, marked as answer]
@Kenny-024 

It's explained here:

https://www.dosbox-staging.org/releases/release-notes/0.81.0/#log-window-support-on-macos

I highly recommend following the Getting Started guide which will give you further information on how to use these startup icons, and show you the best way to set up DOS games on your Mac with DOSBox Staging:

https://www.dosbox-staging.org/getting-started/introduction/
